openfish-webapp/index.html

```diff
@@ -6,8 +6,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>OpenFish</title>
     <link rel="stylesheet" href="./src/index.css" />
+    <script type="module" src="/src/annotation-card.ts"></script>
+    <script>
```

Comment:
What is this script and does it need to be committed?

Reply:
This is just to test the web component to see if it is working.